getting_started_rosa/basic-concepts-rosa.adoc

```diff
@@ -0,0 +1,8 @@
+[id="understanding-rosa"]
+= Understanding {product-title}
```

Comment:
This title should more closely match the title used for the left-nav from `_topic_map.yml`, e.g. "Basic concepts".

Reply:
I'd also update the `id` above it in that case.

Reply:
I think I've got this right now.

Reply:
We're going to clean these up later to be in sync with OSD rewrites. This will happen post-beta.